kusto/kcsb.go

```diff
@@ -23,6 +24,8 @@ type ConnectionStringBuilder struct {
 	ApplicationToken                 string
 	AzCli                            bool
 	MsiAuthentication                bool
+	WorkloadAuthentication           bool
+	FederationTokenFile              string
```

Comment:
Is this the file content or the file path?
if the latter, please add "Path" to the member name.

Reply:
done